sdk/examples/xo_python/sawtooth_xo/processor/handler.py

```diff
@@ -38,88 +39,66 @@ def family_versions(self):
 
     @property
     def namespaces(self):
-        return [self._namespace_prefix]
+        return [xo_state.XO_NAMESPACE]
 
     def apply(self, transaction, context):
 
-        # 1. Deserialize the transaction and verify it is valid
```

Comment:
I realize there is going to be accompanying documentation, but it still seems to me like it would be a good idea to keep some narrative comments in the code (they don't have to be the existing ones).

Reply:
I would like the code to be as clear as possible so that comments are almost not needed. The accompanying documentation can call out particular steps. I would wonder if particular method names could be extended to be clear about what they do. XoPayload.from_bytes() could be something else more descriptive or there could be a comment that it also validates the payload.